get-started/versioning-availability.md

```diff
@@ -108,4 +110,60 @@ deployment:
 
 :::{tip}
 For contributors and those interested in the technical details, see the [Elastic docs syntax guide](https://elastic.github.io/docs-builder/syntax/applies/) for more information on how these badges are implemented.
-:::
\ No newline at end of file
+:::
+
+## Find docs for your product version
+
+In April 2025, we released our new documentation site. This site includes documentation for our latest product versions.
+
+You can find documentation for previous product versions on our [Previous versions](/archive.md) page. You can also access the previous version of a specific page, where available, by clicking the **View previous version** link in the sidebar. 
+
+% not sure about curator
```

Comment:
does this need a "The following products versions are documented on this site:" ?